### ISSUE TITLE
Missing availability of aria-placeholder property with combobox role

### ISSUE THREAD

**Issue by MarioBatusic** [w3c/aria#2689]
Just as using the `textbox` role in the absence of a `<input>` element makes a `<div>` element with `contenteditable` an input for ATs, the same applies to the `combobox` role in the last spec version: 

Encoded on a `<div>` or `<span>` element with `contenteditable`, this role also signals to ATs that it is an input with additional values offered for selection.

Since the HTML `placeholder` attribute is not available in this case, the `aria-placeholder` should be available on elements with the `combobox` role.

**Comment by scottaohara** [comment]
wonder if it would make sense to allow the attribute only in situations where the combobox actually allows text input?  so, if a combobox has been provided an `[aria-autocomplete](https://w3c.github.io/aria/#aria-autocomplete)` attribute, then a placeholder would make sense.  but if it's a select-only combobox, then maybe it shouldn't have a placeholder since it doesn't really have a purpose in that use case / people would be most likely to use it to state something like "choose a value"?

**Comment by jnurthen** [comment]
@MarioBatusic would you like to create a PR for this?

**Comment by scottaohara** [comment]
quickly testing with chromium per the comment in the WG meeting today that placeholder isn't respected on an element with 
`role=combobox`

if the element with the explicit or implicit role=combobox is an input, the placeholder attribute will be exposed.

if the element with an explicit role=combobox is not an input (e.g., it's a button), then the placeholder is not exposed (though it is noted as existing in the a11y properties the dev tools displays)

```
<input list=d placeholder=foo aria-label=test>
<datalist id=d>
	<option>glarp</option>
</datalist>

<hr>

<button role=combobox aria-label=test aria-placeholder=foo aria-expanded=false>
	<span aria-hidden=true>boop</span>
</button>

<hr>

<input role=combobox placeholder=foo aria-expanded=false **aria-label=test>**
```
